packages/babel-plugin/src/__tests__/index.test.ts

```diff
@@ -65,6 +65,11 @@ describe('babel plugin', () => {
       const _ = "._1wyb1fwx{font-size:12px}";
       const MyDiv = forwardRef(
         ({ as: C = "div", style: __cmpls, ...__cmplp }, __cmplr) => {
+          if (__cmplp.innerRef) {
+            throw new Error(
+              "Please use 'ref' instead of 'innerRef'. For more details, visit go/compiled-innerref"
```

Comment:
Ah people outside of atlassian can't access `go/` links (or access our Confluence), so might be best to remove the last sentence